roles/download/defaults/main.yml

```diff
@@ -310,9 +316,14 @@ kubeadm_checksums:
 etcd_binary_checksums:
   # Etcd does not have arm32 builds at the moment, having some dummy value is
   # required to avoid "no attribute" error
-  arm: 0
-  arm64: 444e10e6880595d75aaf55762901c722049b29d56fef50b2f23464bb7f9db74d
-  amd64: 864baa0437f8368e0713d44b83afe21dce1fb4ee7dae4ca0f9dd5f0df22d01c4
+  arm:
+    v3.5.1: 0
```

Comment:
No 3.5.0 arm ?

Reply:
That is a nice point.
The lack can hurt the situation of Kubernetes  v1.22 on arm.

Reply:
Fixed. I'm really curious if we have actual arm 32 bit users and what their experience is with kubespray :)